tests/unit/test_kokkos_serialize_dynrankview.cc

```diff
@@ -59,10 +66,40 @@ struct KokkosDynRankViewTest2D : KokkosViewTest<ParamT> { };
 template <typename ParamT>
 struct KokkosDynRankViewTest3D : KokkosViewTest<ParamT> { };
 
+TYPED_TEST_CASE_P(KokkosDynRankViewTestEmpty);
+TYPED_TEST_CASE_P(KokkosDynRankViewTest0D);
 TYPED_TEST_CASE_P(KokkosDynRankViewTest1D);
 TYPED_TEST_CASE_P(KokkosDynRankViewTest2D);
 TYPED_TEST_CASE_P(KokkosDynRankViewTest3D);
 
+TYPED_TEST_P(KokkosDynRankViewTestEmpty, test_empty_any) {
+  using namespace checkpoint;
+
+  using DataType = TypeParam;
+  using ViewType = Kokkos::DynRankView<DataType>;
+
+  ViewType in_view{};
+  auto out_view = serializeAny<ViewType>(in_view);
+  EXPECT_EQ(out_view->rank(), unsigned(0));
```

Comment:
Shouldn't this also check `size() == 0` - i.e. that it didn't end up looking initialized?

Reply:
```suggestion
  EXPECT_EQ(out_view->rank(), unsigned(0));
  EXPECT_EQ(out_view->size(), unsigned(0));
```